pspamm/scripts/max_hsw.py

```diff
@@ -17,4 +17,4 @@ def getBlocksize(m , n, bk):
 
 def HSW_condition(bm, bn, bk):
 	v_size = 4
-	return (bn + bk) * (bm / v_size) + bn * bk + 2 <= 16
+	return (bn + bk) * (bm / v_size) + bn * bk <= 16
```

Comment:
ceiling division?

Reply:
Similar to `scripts/max_arm.py` where `bm` a multiple of `v_size`.